commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AddAuditLogSyslogHandler.java

```diff
@@ -47,7 +80,17 @@ protected AddAuditLogSyslogHandler(TcpBuilder builder) {
     }
 
     private AddAuditLogSyslogHandler(TlsBuilder builder) {
-        super(builder);
+        this.name = builder.name;
+        this.appName = builder.appName;
+        this.facility = builder.facility;
+        this.formatter = builder.formatter;
+        this.maxFailureCount = builder.maxFailureCount;
+        this.maxLength = builder.maxLength;
+        this.syslogFormat = builder.syslogFormat;
+        this.truncate = builder.truncate;
+        this.replaceExisting = builder.replaceExisting;
+        this.host = builder.host;
+        this.port = builder.port;
```

Comment:
Ah, true, the fun with definite assignment rules :-( I think 3 different constructors, each for one builder, are better, even with slight code duplication.